### PR TITLE
Garnett: new favicons

### PR DESCRIPTION
**Pull request by SiAdcock**
## What does this change?

Garnett related favicons

### CODE REVIEW

File: applications/app/templates/webAppManifest.scala.js [hunk 1]
```diff
@@ -1,34 +1,71 @@
...
+            "sizes": "152x152",
+            "src": "@{JavaScript(Static("images/favicons-garnett/roundel-152x152.png"))}"
+        },
+        {
+            "type": "image/png",
+            "sizes": "114x114",
+            "src": "@{JavaScript(Static("images/favicons-garnett/roundel-114x114.png"))}"
+        }
+    ],
+        "theme_color": "#005689",
```
Comment (sndrs) [inline applications/app/templates/webAppManifest.scala.js]: theme color is old guardian blue, do we need to change that?

Comment (sndrs) [reply applications/app/templates/webAppManifest.scala.js]: (i think it's the colour of tabs in android?)

Comment (sndrs) [reply applications/app/templates/webAppManifest.scala.js]: ah more than that https://developers.google.com/web/updates/2015/08/using-manifest-to-set-sitewide-theme-color

File: common/app/views/fragments/metaData.scala.html [hunk 2]
```diff
@@ -50,9 +59,17 @@
 @* Additional meta data that does not impact rendering speed (and can live at the end of the <head>) *@
 
 @if(context.environment.mode == Dev){
-    <link rel="shortcut icon" type="image/png" href="@Static("images/favicons/32x32-dev.ico")" />
+    @if(experiments.ActiveExperiments.isParticipating(experiments.Garnett)) {
+        <link rel="shortcut icon" type="image/png" href="@Static("images/favicons-garnett/32x32-dev.ico")" />
```
Comment (sndrs) [inline common/app/views/fragments/metaData.scala.html]: nice!

**Review by sndrs — APPROVED** [review]
looks great apart from comments! (if they're relevant)

File: applications/app/templates/webAppManifest.scala.js (hunk 1 above)
Comment (NataliaLKB) [reply applications/app/templates/webAppManifest.scala.js]: Definitely lets change this! It doesn't looks right with the new styles anymore.

Comment (sndrs) [reply applications/app/templates/webAppManifest.scala.js]: white with black text?

Comment (NataliaLKB) [reply applications/app/templates/webAppManifest.scala.js]: Or the grey/blue of the header. We can ask @blongden73 and @zeftilldeath and change tomorrow

**Pull request merged**